app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -203,6 +206,19 @@ public void setUsesCustomExportSchedule(boolean usesCustomExportSchedule) {
     }
 
     /** {@inheritDoc} */
+    @DynamoDBTypeConverted(converter = DynamoUploadSchema.FieldDefinitionListMarshaller.class)
+    @Override
+    public List<UploadFieldDefinition> getUploadMetadataFieldDefinitions() {
+        return uploadMetadataFieldDefinitions;
```

Comment:
Prevent return of null?

Reply:
Changed to look like the other collections in this class.